src/style.css

```
@@ -1 +1,69 @@
-/* Styles go here */
+html * {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
```

```suggestion
html {
  font-family: "Roboto", sans-serif;
  font-size: 12px;
}
```
Обнуляти марджини і падінги краще для тих тегів, де тобі це потрібно, а не для всієї сторінки